h2o-genmodel/src/main/java/hex/genmodel/AbstractMojoWriter.java

```diff
@@ -184,6 +185,7 @@ private void addCommonModelInfo() throws IOException {
     writekv("prior_class_distrib", Arrays.toString(model.priorClassDist()));
     writekv("model_class_distrib", Arrays.toString(model.modelClassDist()));
     writekv("timestamp", model.timestamp());
+    writekv("multiline_categoricals", true);
```

Comment:
How about "escape_domain_values"? It would be good to capture how are we actually treating the new lines in the name of the property.

Reply:
Why not ? Renamed.